src/task.js

```diff
@@ -0,0 +1,35 @@
+import './style.css';
+import { tasks, task } from './tasks.js';
+
+let getTask;
```

Comment:
- [OPTIONAL] Prefer naming your variable the simple way that they reveal the intention behind it, and use a `verb` or a `phrase` for functions so that they says what they do. Here for a verb for variable name and it is a bit confusing (check lines 13 and 15).